website/content/docs/secrets/databases/index.mdx

```diff
@@ -259,6 +345,26 @@ address these challenges, you can configure one or more of the following options
   its passwords, and enables multiple clients using the same static role to
   transition slowly.
 
+</Tab>
+
+<Tab heading="GUI" group="gui2">
+
+![Partial screenshot of the Vault GUI showing the database level "Rotate static roles immediately" toggle](/img/gui/databases/databaseRotation.png)
+On the DB connection creation page, this toggle button will set the default value for static roles whether to rotate immediately or to skip the initial rotation if toggled off.
+
+![Partial screenshot of the Vault GUI showing the role level "Rotate immediately" toggle](/img/gui/databases/roleRotation.png)
+On the static role creation page, this toggle button will control whether this individual role password will be rotated immediately or skipped if toggled off.
```

Comment:
```suggestion
![Partial screenshot of the Vault GUI showing the database level "Rotate static roles immediately" toggle](/img/gui/databases/databaseRotation.png)

On the DB connection creation page and the static role creation page, turning
off the **Rotate static roles immediately** toggle lets you bypass an initial rotation
on role creation.
```

Since the toggle does the same thing, we can simplify the information by explaining the toggle ones and noting that it appears on two different pages